docs/docs/using-pants/advanced-target-selection.mdx

```diff
@@ -185,3 +185,58 @@ For other goals, you can leverage shell piping to partition the input targets in
 ```bash
 pants list :: | awk 'NR % 5 == 0' | xargs pants package
 ```
+
+## Referring to a group of targets
+
+If setting tags on individual targets is not feasible, there are a few other options available to refer to multiple targets.
+
+If you have an operation that you perform often on a certain group of targets, you can use the
+[cli](https://www.pantsbuild.org/2.19/reference/subsystems/cli) subsystem options to create shortcuts. For instance,
+this alias
+
+```toml title="pants.toml"
+publish-libraries = "--filter-target-type=python_distribution --filter-address-regex=\"['^src/libA/,^src/libB/']\" publish src::"
+```
+
+would you let run `pants publish-libraries` to publish all Python distributions declared in the `src/libA` and `src/libB`
+directories.
```

Comment:
What do you think about framing this part as more explicitly as "using CLI aliases"? I think the text is pretty good, I'd just change the heading and maybe also explicitly mention that:

- any arguments/goals can be used
- also that the alias doesn't need to be a "full" invocation, e.g. it can just be a combination of `--filter-...` and `--changed-since` arguments

Reply:
Great, I've renamed and extended the section keeping it where it currently is.